package.json

```diff
@@ -18,9 +18,9 @@
   },
   "devDependencies": {
     "babel-cli": "^6.26.0",
-    "babel-jest": "^21.2.0",
+    "babel-jest": "22.4.3",
```

Comment:
I think we should either nail every dependency or use carrots all the way, this seems inconcise